src/nepiada/utils/noise.py

```diff
@@ -134,27 +134,58 @@ def add_noise(self, data):
     def get_name(self):
         return "Laplacian noise"
 
+class RandomizeData(AdversarialNoiseStrategy):
+    """
+    The algorithm replaces values with random datapoints. This is not standard noise as we DO NOT add noise to the 
+    values, rather we replace the values with random numbers.
+    """
+    def add_noise(self, data):
+        noisy_data = {}
+        for key, value in data.items():
+            if value is None:
+                # Skip None values or handle them differently if needed
+                noisy_data[key] = value
+                continue
+            if isinstance(value, (np.ndarray, np.generic)):
+                # Add noise to each element in the tuple
+                noisy_data[key] = np.array([np.random.randint(0, 50, size = val.shape) for val in value if val is not None], dtype=np.float32)
```

Comment:
50 is a hard-coded value, because importing config will lead to a cyclic dependency here. If anyone has better ideas, feel free to suggest here.

Reply:
You could probably just add it as a field to the __init__ method of the class right?